automatic/qtox/qtox.nuspec

```diff
@@ -4,7 +4,7 @@
   <metadata>
     <id>qtox</id>
     <title>qTox</title>
-    <version>1.4.1.1</version>
+    <version>1.7.0</version>
```

Comment:
please don't change the version to anything newer than what is available on chocolatey.org.
Latest version on chocolatey.org is `1.5.1`